docs/index.md

```diff
@@ -0,0 +1,10 @@
+# About NPM Packages
+
+The `npm` plugin extends [pulpcore](https://pypi.python.org/pypi/pulpcore/) to support
+hosting npm packages. This plugin is a part of the [Pulp Project](http://www.pulpproject.org),
+and assumes some familiarity with the [pulpcore documentation](https://pulpproject.org/user/).
+
+If you are just getting started, we recommend getting to know the :doc:`basic
+workflows<workflows/index>`.
```

Comment:
I guess this will not work anymore because we are removing the workflows/index.rst page, right?
```suggestion
If you are just getting started, we recommend getting to know the [basic workflows](site:pulp_npm/docs/user/guides/sync.md).
```

Reply:
Yes, and we can add that new section in the future. Thanks for point it our @git-hyagi